src/uu/fmt/src/fmt.rs

```diff
@@ -1,3 +1,7 @@
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
```

Comment:
Here and in the other `fmt` files there is already a license header.

Reply:
oh yeah, the wording isn't the same :(